test/test.php

```diff
@@ -98,8 +98,10 @@ function () {
         );
 
         $vite->preloadImages();
+        $vite->preloadFonts();
```

Comment:
why preload fonts in this test? the assertion on `$tags->preload` below indicates no fonts are being preloaded.

probably either add something to the manifest to demonstrate preloading of fonts? or remove this call?